### DISCUSSION TITLE
[bug] umi现在支持得很差，打算弃了，转vite

### DISCUSSION THREAD

**Discussion by xijin** [umijs/umi#11794] [General]
我描述一下吧，使用umi 两年多了，陆续碰到了很多问题，基本上都是找到issue一个个看，看了之后也没有发现解决，然后莫名其妙的就关闭了。
包括不限于：
1. umi3 下msfu 开启后和qiankun 子应用冲突，报错。
2. umi3 下splitChunks报错，AssertionError [ERR_ASSERTION]: chunk of antd not found，和文档不一致，看了下讨论，没有解决 https://github.com/umijs/umi/issues/5136
3. umi3 升级umi4之后，api一样，但是功能变了，比如 onRouteChange 实际使用的时候和文档也对不上，官方推荐升级，但是也不了了之。
算了，算了，拜拜了

**Comment by objHua** [comment]
看最近更新的内容和频率就造，后面应该不会再有新feature优化了，都是在维护吃老本了
